transport/src/main/java/io/netty/channel/AbstractChannel.java

```diff
@@ -56,8 +56,6 @@
     private final ChannelId id;
     private final Unsafe unsafe;
     private final DefaultChannelPipeline pipeline;
-    private final ChannelFuture succeededFuture = new SucceededChannelFuture(this, null);
-    private final VoidChannelPromise voidPromise = new VoidChannelPromise(this, true);
```

Comment:
move this to DefaultChannelPipeline as we delegate for other methods into it as well